### PR TITLE
[Improvement][DivTeam] Tracker optimization for *arr suite

### PR DESCRIPTION
**Pull request by jasimancas**
The indexer has been optimized so that the search results in radarr are more efficient:

- The title has been adapted to the "SceneStyle" format used by radarr / sonarr
- Spaces and other characters have been removed
- Tracker language has been added to improve detection (.Spanish)
- The name of the tracker has been added to the end as release group (*-DivTeam)
- Points duplication has been replaced
- The title has been cleaned

The recommendations of the radarr team have been followed to greatly improve the detection, several tests have been carried out and the quality, the language and the number of results it returns have been improved.

### CODE REVIEW

**Comment by garfield69** [comment]
when you are happy you have completed this PR then flick from Draft to ready-for-review.

**Comment by jasimancas** [comment]
> when you are happy you have completed this PR then flick from Draft to ready-for-review.
Yes, but I want to fine tune the indexer as much as possible, but I have a doubt, how do I make an "and" in these two sentences?
(a:not(:contains("VOSE")) AND (a:contains("MHD1080"),a:contains("m1080p"),a:contains("MicroHD 1080p")))

I have obtained the "or" using the comma, but it has to be fulfilled that it does not contain VOSE and that it contains some of the 3 following ones to adjust the qualities and to facilitate the work to the arr

File: src/Jackett.Common/Definitions/divteam.yml [hunk 1]
```diff
@@ -173,4 +179,3 @@ search:
         img[src="images/4x.png"]: 4
         img[src="images/5x.png"]: 5
         "*": 1
-# xbtit customised
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/divteam.yml:176]: please leave the engine tags in place.
they are useful when looking for similar indexers to compare or to use as a template for a request to add a new indexer that uses the same search engine.

Comment (jasimancas) [reply src/Jackett.Common/Definitions/divteam.yml]: Sorry.
Added.

**Review by garfield69 — APPROVED** [review]

**Pull request merged**